packages/components/src/Popover/types.ts

```diff
@@ -0,0 +1,124 @@
+import { CSSProperties, PropsWithChildren } from "react";
```

Comment:
See https://github.com/GetJobber/atlantis/pull/2451/files#r2025090611
But I do love the use of types files